spk/duplicity/src/requirements.txt

```diff
@@ -0,0 +1,21 @@
+Babel==1.3
+debtcollector==0.5.0
+duplicity==0.7.03
+iso8601==0.1.10
+lockfile==0.10.2
+netaddr==0.7.14
+netifaces==0.10.4
```

Comment:
Suggestion: remove the versions for wheels that you create (and move to top or bottom of the file, to distinguish them easier, and you can even add a comment). Othrwise, when the cross package is updated and this requirements file is overlooked, the package won't compile anymore.